templates/pkg/equal/struct.go.tpl

```diff
@@ -0,0 +1,19 @@
+{{ template "boilerplate" }}
+
+package equal
```

Comment:
After further thought, I think it would be best to name the package `pkg/compare` and have the functions comparing struct equality be named `IsEqual{ShapeName}`.

We can then alias `pkg/compare` to `svccompare` and then in the `pkg/resource/{resource}/delta.go` file's `newDelta` function, we would use both `ackcompare` general utility comparison functions as well as `svcompare` service-type-specific comparison functions.

The reason I would like to name the package `pkg/compare` instead of `pkg/equal` is because I think eventually we will want to generate comparison functions that evaluate more than just equality. For instance, we might have comparison functions that return whether an enum status string field has one of a set of valid values...

Reply:
++ for `pkg/compare` and import as `svccompare`